phfs/phfs.h

```diff
@@ -19,6 +19,10 @@
 #include <devices/devs.h>
 #include <hal/hal.h>
 
+/* phfs_open flags */
+#define PHFS_OPEN_RAWONLY 0x80000000
```

Comment:
I think that there is no need to create additional define to point to the same attributes as the following enum:
https://github.com/phoenix-rtos/plo/blob/6bd8f71a3f6e27c0d28184ca4216a8d9b1255db4/phfs/phfs.c#L27

I would suggest to put the `enum` in header and use it instead of defining the new fields. On the other hand, you can use defines instead of `enum` (remove it)  in the whole `phfs` unit.

Reply:
> I think that there is no need to create additional define to point to the same attributes as the following enum:
> 
> https://github.com/phoenix-rtos/plo/blob/6bd8f71a3f6e27c0d28184ca4216a8d9b1255db4/phfs/phfs.c#L27
> 
> I would suggest to put the `enum` in header and use it instead of defining the new fields. On the other hand, you can use defines instead of `enum` (remove it) in the whole `phfs` unit.

From what I see phfs_prot_* is used internally along with phfs_* structures defined in the phfs.c file, while the public API defined in phfs.h provides phfs_open() with parameter flags. I believe the purpose of flags is to tell phfs_open() how it should behave (rather than just forward these flags to phoenixd on the remote side).

Reply:
Added separate commit which replaces enums with defines in the internal API.